### PR TITLE
Control progress bar

### PR DESCRIPTION
**Pull request by lekaverta**
Importing leads we want to use the same progress to do upload to s3 the convertion XLS to CSV to looks like just one operation. So we limit the max progress from bar and get the data object to call data.context.remove in our s3_uploads_complete.

### CODE REVIEW

**Comment by lekaverta** [comment]
[[XLS - Import] Atualização na interação do usuário durante o upload do arquivo](https://trello.com/c/i5JD5eI7/243-xls-import-atualiza%C3%A7%C3%A3o-na-intera%C3%A7%C3%A3o-do-usu%C3%A1rio-durante-o-upload-do-arquivo)

File: app/assets/javascripts/s3_direct_upload.js.coffee [hunk 1]
```diff
@@ -92,10 +93,10 @@ $.fn.S3Uploader = (options) ->
               return event.result
 
         data.context.remove() if data.context && settings.remove_completed_progress_bar # remove progress bar
-        $uploadForm.trigger("s3_upload_complete", [content])
+        $uploadForm.trigger("s3_upload_complete", [content, data])
```
Comment (fccoelho7) [inline app/assets/javascripts/s3_direct_upload.js.coffee]: Why did you remove data property? Don't we need it in anywhere?

Comment (fccoelho7) [inline app/assets/javascripts/s3_direct_upload.js.coffee]: Why did you remove data property? Don't we need it in anywhere?

Comment (lekaverta) [reply app/assets/javascripts/s3_direct_upload.js.coffee]: No, we added data property. We need this element to control progress bar in our s3_upload_complete function.

Comment (fccoelho7) [reply app/assets/javascripts/s3_direct_upload.js.coffee]: Oh! My bad. No problem. #lol
